src/probnum/quad/solvers/policies/_max_aquisition_policy.py

```diff
@@ -0,0 +1,71 @@
+"""Max acquisition policy for Bayesian quadrature."""
+
+from __future__ import annotations
+
+from typing import Optional
+
+import numpy as np
+
+from probnum.quad.solvers._bq_state import BQState
+from probnum.quad.solvers.acquisition_functions import AcquisitionFunction
+from probnum.typing import IntLike
+
+from ._policy import Policy
+
+# pylint: disable=too-few-public-methods, fixme
+
+
+class RandomMaxAcquisitionPolicy(Policy):
+    """Policy that maximizes an acquisition function by sampling random candidate nodes.
+
+    The candidate nodes are random draws from the integration measure. The node with the
+    largest acquisition value is chosen.
+
+    Parameters
+    ----------
+    batch_size
+        Size of batch of nodes when calling the policy once (must be equal to 1).
```

Comment:
Just to confirm I'm thinking this correctly: If we returned `n = batch_size > 1` nodes by taking from the random sample those `n` nodes at which the acquisition function takes its `n` largest values, we would likely end up with a bunch of nodes very close to one another, which is not very useful.

Reply:
Yes, exactly :)

Reply:
There are ways around it e.g., by penalizing areas around points (e.g. this https://arxiv.org/pdf/1812.01553v1.pdf)